automated-comments/go/hamming/use_increase_operator.md

```diff
@@ -0,0 +1 @@
+Go has a special `++` operator for incrementing a variable, like this: `count++`.
```

Comment:
Here the context and recommendation is implied. I think I'd lean towards being explicit:

> In most cases, Go programmers would use Go's `++` operator to increment. E.g. `count++`.